Gemfile

```diff
@@ -37,6 +37,9 @@ gem "rack-attack", "~> 6.5"
 gem "net-smtp", "~> 0.3.1"
 gem "psych", "~> 4.0"
 
+gem "pronto", "~> 0.11"
+gem 'pronto-rubocop', require: false
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.